docs/fundamentals/productivity/configure-code-analysis-rules.md

```diff
@@ -28,8 +28,8 @@ The following table shows the different rule severities.
 |-|-|
 | `error` | Violations appear as build *errors* and cause builds to fail.|
 | `warning` | Violations appear as build *warnings* but do not cause builds to fail (unless you have an option set to treat warnings as errors). |
-| `suggestion` | Violations appear as build *messages* and as suggestions in the Visual Studio IDE. |
-| `silent` | Violations aren't visible to the user. |
+| `suggestion` | Violations appear as build *messages* and as suggestions in the Visual Studio IDE. Suggestions appear as three gray dots under the first two characters in Visual Studio. |
+| `silent` | Violations aren't visible to the user. Code generation features generate code in this style, however. Rules with silent severity participate in cleanup and appear in the Quick Actions and Refactorings menu. |
```

Comment:
> Code generation features generate code in this style, however.

This seems unclear to me.

Reply:
> Rules with silent severity participate in cleanup and appear in the Quick Actions and Refactorings menu.

There is no context for `cleanup` here.

Reply:
Seems the table content should match: https://docs.microsoft.com/visualstudio/code-quality/use-roslyn-analyzers?view=vs-2019#rule-severity

Reply:
@mavasani This is a copy of what in VS docs, so I'm not 100% certain. But I think it says that auto-generated code respects the "suggestion" severity.

Reply:
Since this is in the .NET docs and not VS, the severities shouldn't be tooling focused. Perhaps just mention that to see how they affect Visual Studio features, see [link to VS page].

Reply:
Good point @gewarren!

Reply:
> 
> 
> > Rules with silent severity participate in cleanup and appear in the Quick Actions and Refactorings menu.
> 
> There is no context for `cleanup` here.

So cleanup is just incorrect from VS docs?

Reply:
> 
> 
> Since this is in the .NET docs and not VS, the severities shouldn't be tooling focused. Perhaps just mention that to see how they affect Visual Studio features, see [link to VS page].

Should I then revert my changes and add "For more information, see ......"?

Reply:
@Youssef1313 I specifically didn't add the VS effects since this is the .NET docs, but I do think it's a good idea to maybe add a TIP box under the table to say "For information about how rule severities surface in Visual Studio, see []()."

Reply:
@gewarren I updated the PR per your suggestion.